.rubocop.yml

```diff
@@ -80,3 +80,6 @@ Style/RedundantRegexpEscape:
 
 Style/SlicingWithRange:
   Enabled: true
+
+Gemspec/DevelopmentDependencies:
+  EnforcedStyle: gemspec
```

Comment:
This has been addressed in `main`. Can you please rebase and revert this change out

Reply:
I've done rebase and removed changes in .rubocop.yml